rfcs/formats/communication/relay.md

```diff
@@ -8,7 +8,78 @@
 - RFC: 03-015
 - Authors: Venkatraman Ramakrishna, Krishnasuri Narayanam, Sandeep Nishad, Dhinakaran Vinayagamurthy
 - Status: Proposed
-- Since: 02-Apr-2022
+- Since: 30-May-2022
 
 ## Summary
 
+
+## NetworkQuery
+
+This message is created by client application to encapsulate the query for the relay.
+Whenever a client application needs to query a foreign network for a [ledger state view](../../models/ledger/views.md), it must wrap the query comprising of a [view address](../views/addressing.md), a [verification policy](../policies/proofs-verification), and other metadata, into a DLT-neutral structure as follows for the [relays](../../models/infrastructure/relay.md) that will accept and route the request.
+
+```protobuf
+message NetworkQuery {
+  repeated string policy = 1;
+  string address = 2;
+  string requesting_relay = 3;
+  string requesting_network = 4;
+  string certificate = 5;
+  string requestor_signature = 6;
+  string nonce = 7;
+  string requesting_org = 8;
+}
+```
+
+* `policy`: specifies the [verification policy](../policies/proofs-verification) to be used for this query.
+* `address`: stores the view address. More on addressing [here](../views/addressing).
+* `requesting_relay`: id for the local relay (Optional, by default is filled by the relay to which this message is sent). 
+* `requesting_network`: id for the local network to which this client application belongs.
+* `certificate`: X509 Certificate in PEM format of the requesting client (to authenticate membership of the client to the requesting network, and also used for access control).
+* `requestor_signature`: Digital signature over concatenation of `address` and `nonce` by client, to ensure that the query has not been tampered with.
+* `nonce`: used to avoid replay attacks by entities masquerading as an application.
+* `requesting_org`: Organization id to which this client application belongs.
+
+**Example:**
+
+```
+NetworkQuery { 
+  policy: ["CarrierMSP", "SellerMSP"], 
+  address: "bc-interop-wtln-test.sl.cloud9.ibm.com:9089/wtln/wtln-channel:wtlncc:GetBillOfLadingByPurchaseOrder:PO59133361", 
```

Comment:
Can you change the hostname to something neutral (without an "ibm.com")? Say "trade-logistics-relay.com"?